reference-implementation/lib/helpers.js

```diff
@@ -62,6 +62,10 @@ exports.IsFiniteNonNegativeNumber = v => {
 };
 
 exports.InvokeOrNoop = (O, P, args) => {
+  assert(exports.typeIsObject(O) === true);
+  assert(typeof P === 'string');
+  assert(Array.isArray(args));
```

Comment:
Yeah, although in the spec it still has a "If args was not passed, let args be a new empty List" line which is I guess now unnecessary. We should be sure to remove that.